spec/views/rb_taskboards/show_spec.rb

```diff
@@ -250,7 +255,7 @@
 
       render
 
-      assert_select ".model.work_package.impediment" do |impediment|
+      assert_select '.model.work_package.impediment' do |impediment|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.